packages/gatsby-theme-project-portal/src/components/Cards.stories.tsx

```diff
@@ -5,22 +5,25 @@ import * as CardStories from "./Card.stories"
 const meta: Meta<typeof Cards> = {
   component: Cards,
   tags: ["autodocs"],
+  includeStories: /^[A-Z]/,
```

Comment:
why do you need this here and not in the other stories?

Reply:
That line is necessary whenever you have a variable (like `exampleCards`) which shouldn't be a `Story`. Storybook looks for exported variables in the file and creates stories from them, so if you have other variables you have to tell it which ones are meant to be stories and which not. The recipe comes from here: https://storybook.js.org/docs/react/api/csf#non-story-exports